src/x86/bindings.rs

```diff
@@ -1121,47 +1070,38 @@ pub struct kvm_ioapic_state {
     pub pad: __u32,
     pub redirtbl: [kvm_ioapic_state__bindgen_ty_1; 24usize],
 }
+
 #[repr(C)]
 #[derive(Copy, Clone)]
 pub union kvm_ioapic_state__bindgen_ty_1 {
     pub bits: __u64,
     pub fields: kvm_ioapic_state__bindgen_ty_1__bindgen_ty_1,
 }
 
-// Manual implementation of `Versionize` since `versionize_derive` doesn't support unions
-// anymore. What we're doing here is equivalent to the old autogenerated implementation, namely
-// we're creating a bitwise serialization of the object.
-impl Versionize for kvm_ioapic_state__bindgen_ty_1 {
-    fn serialize<W: std::io::Write>(
-        &self,
-        writer: &mut W,
-        version_map: &VersionMap,
-        app_version: u16,
-    ) -> VersionizeResult<()> {
-        // Safe because this is a POD object and we're only using the read result to serialize
-        // the underlying bytes.
+impl Serialize for kvm_ioapic_state__bindgen_ty_1 {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
         let bits = unsafe { self.bits };
```

Comment:
Could we preserve the safety comment here?

Reply:
oh, shoot. Missed that, thanks!

Reply:
fixed